src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -182,13 +183,33 @@ class ConnectionHandlerImpl
    * \param hash_id pointer for session hash identifier
    * \return uint32_t Id (number) of new session if successful, otherwise 0.
    */
+  // DEPRECATED
   virtual uint32_t OnSessionStartedCallback(
       const transport_manager::ConnectionUID connection_handle,
       const uint8_t session_id,
       const protocol_handler::ServiceType& service_type,
       const bool is_protected,
       uint32_t* hash_id);
```

Comment:
I guess the method is in master branch as well.
If I removed this, I also needed to remove old ProtocolHandlerImpl::HandleControlMessageStartSession() to avoid build error. So maybe I should keep this for now?